src/app/about/about.component.spec.ts

```diff
@@ -20,6 +20,7 @@ import { AdvancedsearchComponent } from '../advancedsearch/advancedsearch.compon
 import { SearchBarComponent } from '../search-bar/search-bar.component';
 import { ContactComponent } from '../contact/contact.component';
 import { ModalComponent } from 'ng2-bs3-modal/ng2-bs3-modal';
+import { TopbarComponent } from '../topbar/topbar.component';
```

Comment:
but I don't find making a separate component as a good solution.try to resolve the errors you face.